pkg/api/types.go

```diff
@@ -683,6 +683,14 @@ const (
 	AffinityTypeNone AffinityType = "None"
 )
 
+// LoadBalancerInfo defines whether the LoadBalancer is rewriting or
+// pass through
+type LoadBalancerInfo struct {
+	DestIP   string `json:"destIP,omitempty" description:"External IP of the Load Balancer"`
+	SourceIP string `json:"sourceIP,omitempty" description:"Source IP of the Load Balancer"`
```

Comment:
Ahh, I see.  The IP of packets from the LB is not even the LB's public IP.  Double puke.  This makes me unhappy.  I don't have a better answer on hand, but I really want to think about this more.  What would you say to making two PRs out of this - one for all your support EXCEPT public LB, and a second for just that part?